packages/lab/src/date-picker/DatePicker.tsx

```diff
@@ -103,6 +103,7 @@ export interface DatePickerProps
    * Callback fired when the input value change.
    */
   onChange?: ChangeEventHandler<HTMLInputElement>;
+  compact?: boolean;
```

Comment:
boolean props offer you two choices, on or off. 
If the requirements change, we only have 2 choices, which can lead to breaking changes ?

So I am always wary of changes with boolean props.
Is it something that is truly binary or do we just have 2 options at the moment ?
Also does the prop sufficiently define the behaviour ? 
Compact could be anything to be honest ?

I think this is a good example of where our choice of API can help improve the understanding of the behaviour as well as create a scalable API that is not limited to 2 choices (binary).

`visibleMonths = 1|2`

would be more scalable